oomph-vscode/syntaxes/oomph.tmLanguage.json

```diff
@@ -37,9 +37,30 @@
       ]
     },
     "strings": {
-      "name": "string.quoted.double.oomph",
-      "begin": "\"",
-      "end": "\"",
+      "patterns": [
+        {
+          "name": "string.quoted.triple.oomph",
+          "begin": "\"{3}",
+          "end": "\"{3}|{",
+          "patterns": [
+            {
+              "include": "#string_guts"
+            }
+          ]
+        },
+        {
+          "name": "string.quoted.double.oomph",
+          "begin": "\"|}",
+          "end": "\"|{",
```

Comment:
How does this higlight `"foo {a} bar {b} baz"`?  The pygments lexer treats `} bar {` as a multiline string, which isn't perfect but works fine in practice.

Reply:
Like this:
![Code Screenshot](https://i.vgy.me/vddUgw.png)